### ISSUE TITLE
Last song have different file permissions than others

### ISSUE THREAD

**Issue by olehj** [bleskodev/rubyripper#615]
I just noticed today that the last song in a rip session has the "rw" flag for owners, but rest have "rw-rw-r" which is rather preferred. I use the -Z flag if that should be a cause to it, and also rip to FLAC.

Using 0.7.0rc1

**Comment by Thovthe** [comment]
Could you try again with RC2 and see if the behaviour is still there?

**Comment by olehj** [comment]
No, still just -rw------- on the last, rest has -rw-rw-r-- as expected. Almost looks like the permission gets set for the "previous file" but are skipped in a loop for the last one.

**Comment by bleskodev** [comment]
Hi, interesting problem, I cannot reproduce it however, all my files (also FLAC) have -rw-r--r-- permissions, which sounds reasonable to me.

**Comment by RealBrewer** [comment]
Adding my 2 cents here. I rip to both flac and mp3 and I noticed that this occurs ONLY with the last flac track, but not the last mp3, if that observation is of any use. (rc4 BTW)
